vm-virtio/src/net.rs

```diff
@@ -538,7 +536,7 @@ impl VirtioDevice for Net {
     fn activate(
         &mut self,
         mem: GuestMemoryMmap,
-        interrupt_evt: EventFd,
+        interrupt_evt: Arc<Box<Fn(u16) + Send + Sync>>,
```

Comment:
We may want to rename all those `interrupt_evt` as `interrupt_cb` now.

Reply:
Sounds good!